### PR TITLE
 Layout reset password

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
 Layout reset password and create table password reset
## Reference Requirements
- Task ID: #191

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/ResetPasswordController.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+     *
+     * @param \Illuminate\Http\Request $request request
+     * @param string|null              $token   reset token
+     *
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function showResetForm(Request $request, $token = null)
+    {
+        return view('user.auth.resetpassword')->with(
+            ['token' => $token, 'email' => $request->email]
```
Comment (at-haytran) [inline app/Http/Controllers/ResetPasswordController.php]: ([
   'token' => $token,
   'email' => $request->email
])

File: resources/views/user/auth/email.blade.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                        <div class="form-group text-center">
+                            <input type="submit" value="{{ __('resetPassword.button.link_reset') }}" class="btn btn-success">
+                        </div>
+                    </form>     
+                </div>
+            </div>
+        </div>
+    </div>
+@endsection
+
```
Comment (at-haytran) [inline resources/views/user/auth/email.blade.php]: dư 1 dòng

File: resources/views/user/auth/resetpassword.blade.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                        <div class="form-group text-center">
+                            <input type="submit" value="{{ __('resetPassword.reset_password') }}" class="btn btn-success">
+                        </div>
+                    </form>     
+                </div>
+            </div>
+        </div>
+    </div>
+@endsection
+
```
Comment (at-haytran) [inline resources/views/user/auth/resetpassword.blade.php]: dư 1 dòng

**Comment by at-haytran** [comment]
LGTM

**Comment by at-haytran** [comment]
LGTM

File: app/Http/Controllers/ForgotPasswordController.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+use Illuminate\Http\Request;
+
+class ForgotPasswordController extends Controller
+{
+     /**
+     * Display the form to request a password reset link.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function showLinkRequestForm()
```
Comment (at-vietpham) [inline app/Http/Controllers/ForgotPasswordController.php]: Method này dùng để làm gì?
Tham khảo: https://github.com/laravel/framework/blob/5.6/src/Illuminate/Foundation/Auth/SendsPasswordResetEmails.php

File: resources/lang/en/resetPassword.php [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?php
+    
+    return [
+        'forget_password' => 'Forgot Your Password?',
+        'reset_password' => 'Reset Password',
+        'button' => [
+            'link_reset' => 'Send Password Reset Link',
```
Comment (at-vietpham) [inline resources/lang/en/resetPassword.php]: "Send link reset password" chứ

File: resources/views/user/auth/resetpassword.blade.php [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+@section('content')
+    <div class="container">
+        <div class="row">
+            <div class="col-md-6 col-md-offset-3">
+                <div class="contact-wrap">
+                    <h3>{{ __('resetPassword.reset_password') }}</h3>
+                    <form id="loginForm">
+                        <div class="row form-group">
+                            <div class="col-md-12">
+                                <label for="email">{{ __('E-Mail Address') }}</label>
```
Comment (at-vietpham) [inline resources/views/user/auth/resetpassword.blade.php]: Không dùng text lang kiểu này. Tạo key rồi gọi ra như những text khác.
Tương tự những nơi khác

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by at-vietpham** [comment]
LGTM 🖌

**Pull request merged**